src/getHumanAge.test.js

```diff
@@ -4,7 +4,26 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('should return an array', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test('should return 1 year for first 15 years of cat and dog', () => {
```

Comment:
this test ensures that 14 years is not even 1 year for dog and cat
but we cannot be sure that `getHumanAge(15, 15)` will return `[0, 0]`, which says test description

let's fix this case